modules/platforms/nodejs/lib/internal/Router.js

```diff
@@ -0,0 +1,531 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Util = require('util');
+const Errors = require('../Errors');
+const IgniteClient = require('../IgniteClient');
+const ClientSocket = require('./ClientSocket');
+const AffinityAwarenessUtils = require('./AffinityAwarenessUtils');
+const BinaryUtils = require('./BinaryUtils');
+const BinaryObject = require('../BinaryObject');
+const ObjectType = require('../ObjectType');
+const ArgumentChecker = require('./ArgumentChecker');
+const Logger = require('./Logger');
+
+// Number of tries to get cache partitions info
+const GET_CACHE_PARTITIONS_RETRIES = 3;
+// Delay between tries to get cache partitions info
+const GET_CACHE_PARTITIONS_DELAY = 100;
```

Comment:
Is it in millis?

Reply:
Yes, it is. I will add this to the comment for this const.